scripts/package-osx.sh

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
+set -e
```

Comment:
Is the `curl --upload-file "$dmg" https://transfer.sh/` code still used? If so, then this is not safe because curl returns an exit code of 0 for a lot of HTTP status codes a human might consider errors (401, 403, maybe even 502). It's possible to work around that by using `--fail`, but even that's not guaranteed to result in a non-zero exit code. To make that uploads with curl robust, you need `curl --write-out '%{http_code}' and check whatever that writes.

Reply:
https://github.com/metabrainz/picard/pull/1085